### PR TITLE
Integrated playground at tinygo.org/play

### PR DESCRIPTION
**Pull request by aykevl**
Instead of having a separate https://play.tinygo.org, use https://tinygo.org/play that actually integrates into the Docsy theme.

Once this is in, I'd like to make play.tinygo.org just a redirect to tinygo.org/play so that I can modify the default UI.

### CODE REVIEW

**Comment by aykevl** [comment]
This now also supports a 'share' button!

**Comment by aykevl** [comment]
Rebased after #429 and moved some shared code to a file that's included from multiple places (to avoid excessive duplication).

Ready for review :) I'd like to get this merged as it's pretty big (both the integrated playground and the "share" button).

File: content/play/_index.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+					<p>
+						Like the Go Playground, it's possible to share code
+						snippets. These code snippets are stored on Google's
+						servers in the US and can be viewed by some TinyGo
+						members. While we'll try to make sure the shared data
+						can only be accessed by those who have the URL, we
+						cannot guarantee security. If you accidentally shared
+						something that should not be shared (for example, a
+						password or personally identifying information), you can
+						contact us at [TODO] with the URL so we can remove it.
```
Comment (deadprogram) [inline content/play/_index.md]:
```suggestion
						contact us at [conduct@tinygo.org](mailto:conduct@tinygo.org) with the URL so we can remove it.
```

**Review by deadprogram — APPROVED** [review]
:fireworks: this is awesome!

**Comment by aykevl** [comment]
Updated!
It seems my comment was lost. I wonder whether we should maybe use security@ or privacy@ instead of conduct@tinygo.org? It can be a simple alias of course.

File: content/play/_index.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+						Like the Go Playground, it's possible to share code
+						snippets. These code snippets are stored on Google's
+						servers in the US and can be viewed by some TinyGo
+						members. While we'll try to make sure the shared data
+						can only be accessed by those who have the URL, we
+						cannot guarantee security. If you accidentally shared
+						something that should not be shared (for example, a
+						password or personally identifying information), you can
+						contact us at
+						<a href="mailto:conduct@tinygo.org">conduct@tinygo.org</a>
```
Comment (deadprogram) [inline content/play/_index.md]:
```suggestion
						<a href="mailto:privacy@tinygo.org">privacy@tinygo.org</a>
```

Changes to alias `privacy@tinygo.org` as suggested.

Comment (deadprogram) [reply content/play/_index.md]: @aykevl what do you think?

Comment (aykevl) [reply content/play/_index.md]: I've updated the email address!

**Comment by deadprogram** [comment]
This is fantastic @aykevl now merging!

**Pull request merged**